<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServicesWindow</class>
 <widget class="QDialog" name="ServicesWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>493</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="windowTitle">
      <string>ScrollArea</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>463</width>
        <height>463</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="technologiesLabel">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Technologies</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="noTechnologiesLabel">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>No technologies</string>
         </property>
         <property name="indent">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="technologiesLayout">
         <property name="spacing">
          <number>5</number>
         </property>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="servicesLabel">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Services</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="noServicesLabel">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>No services</string>
         </property>
         <property name="indent">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="servicesLayout">
         <property name="spacing">
          <number>5</number>
         </property>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
